tests/extendrtests/src/rust/src/lib.rs

```diff
@@ -1,53 +1,70 @@
 use extendr_api::prelude::*;
 mod submodule;
 use submodule::*;
+use std::collections::HashMap;
 
-/// Return string `"Hello world!"` to R.
-/// @export
+/// @usage NULL
```

Comment:
Not sure I like deleting the actual meaningful comments (such as "Return string ...). You probably don't need to add `@usage NULL` at all, and can instead use `//` rather than `///` to indicate the comment. Then the comment doesn't get moved into the R wrapper, the function is not exported, and R check should be happy.